easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.3.eb

```diff
@@ -103,7 +103,8 @@ full_sanity_check = True
 eggname = 'gc3pie-%%(version)s-py%s.egg' % pyver
 sanity_check_paths = {
     'files': ['bin/gc3utils',
-              ('lib/python%s/site-packages/%s' % (pyver, eggname), 'lib64/python%s/site-packages/%s' % (pyver, eggname))],
+              ('lib/python%s/site-packages/%s' % (pyver, eggname),
+               'lib64/python%s/site-packages/%s' % (pyver, eggname))],
```

Comment:
@migueldiascosta we usually tend to avoid checking for actual `.egg's more recently, since they could be either a file or a directory, depending on a number of things.

In addition, an `import gc3pie` check is already done too.

So, I'd change this to:

```python
('lib/python%s/site-packages' % pyver, 'lib64/python%s/site-packages'),
```

and move it to `dirs`?